.github/workflows/new-meetup.yml

```
@@ -24,6 +24,10 @@ jobs:
 
       - uses: ./.github/actions/setup-and-cache
 
+      - uses: szenius/set-timezone@ce9c440bc3af4f51b28d7ef7e5c47ee8f26c8dcb # tag v1.2
+        with:
+          timezoneLinux: "Europe/Helsinki"
```

Tämä action on loppujen lopuksi aika yksinkertainen. Otetaanko inlineksi tähän vaan tuo komento? https://github.com/szenius/set-timezone/blob/ce9c440bc3af4f51b28d7ef7e5c47ee8f26c8dcb/index.js#L13-L16

```suggestion
      - name: Set time zone
        run: sudo timedatectl set-timezone Europe/Helsinki
```